### PR TITLE
DM-28926: store the config schema as code

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/ts/ATDome/config_schema.py [hunk 1]
```diff
@@ -1,3 +1,29 @@
+# This file is part of ts_ATDome.
+#
+# Developed for the LSST Data Management System.
```
Comment (wvreeven) [inline python/lsst/ts/ATDome/config_schema.py:3]: Please update this and all other files to Vera C. Rubin Observatory.

**Review by wvreeven — APPROVED** [review]
One minor comment but otherwise this looks good.

File: python/lsst/ts/ATDome/config_schema.py [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["CONFIG_SCHEMA"]
+
+import yaml
+
+CONFIG_SCHEMA = yaml.safe_load(
+    """
 $schema: http://json-schema.org/draft-07/schema#
 $id: https://github.com/lsst-ts/ts_ATDome/blob/master/schema/ATDome.yaml
```
Comment (wvreeven) [inline python/lsst/ts/ATDome/config_schema.py:28]: Please update the path to reflect the correct file.

**Review by wvreeven — APPROVED** [review]
Added another minor comment.

File: python/lsst/ts/ATDome/config_schema.py (hunk 2 above)
Comment (r-owen) [reply python/lsst/ts/ATDome/config_schema.py]: Done. Thanks.

File: python/lsst/ts/ATDome/config_schema.py (hunk 1 above)
Comment (r-owen) [reply python/lsst/ts/ATDome/config_schema.py]: Good catch. I did that for all the packages.

**Pull request merged**